pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -22,6 +25,14 @@ type LifecycleProcessor struct {
 	DetectLambdaLibrary func() bool
 }
 
+var traceEnabled, _ = strconv.ParseBool(os.Getenv("DD_TRACE_ENABLED"))
+var managedServiceEnabled, _ = strconv.ParseBool(os.Getenv("DD_TRACE_MANAGED_SERVICES"))
```

Comment:
what's happening if ParseBool is failing?

Reply:
I'll catch the error.